apps/cli/src/commands/run.command.ts

```diff
@@ -102,8 +105,27 @@ export default class RunCommand extends BaseCommand {
 
       ioClient.on('configuration-updated', async (data: Configuration) => {
         Logger.info(
-          `Configuration change received from API (name: ${data.name}, value: ${data.value})`
+          `Configuration change received from API (name: ${data.name})`
         )
+
+        // To test the below condition of quitOnDecryptionFailure, uncomment the below line
+        // data.isPlaintext = false
+
+        if (!data.isPlaintext) {
+          const configurations = await this.fetchConfigurations()
```

Comment:
This isn't required. We are already getting the newly updated configuration in `data`